### ISSUE TITLE
Add hover style for Admin->Permissions table

### ISSUE THREAD

**Issue by lolimay** [RocketChat/Rocket.Chat#13958]
The current  Admin->Permissions table does not have a hover style as shown below:

![without_hover](https://user-images.githubusercontent.com/32427260/55284013-ded6c280-53a0-11e9-8be8-74f51293b4b3.gif)
 
If a hover style can be added, users can see each row more clearly when they are modifying Permissions for roles.

**Comment by Gautime** [comment]
Hey! Is anybody working on this issue?

**Comment by lolimay** [comment]
> Hey! Is anybody working on this issue?

Yes, and I've already pulled the request. See [#13959](https://github.com/RocketChat/Rocket.Chat/pull/13959)
